### DISCUSSION TITLE
Logging level for `openmeteo-api serve`

### DISCUSSION THREAD

**Discussion by drmrbrewer** [open-meteo/open-meteo#749] [General]
Is it possible to set the logging level for `openmeteo-api serve`, between error, warning, info, debug etc?

**Comment by patrick-zippenfenig** [comment]
Environment variable LOG_LEVEL can be set: https://docs.vapor.codes/basics/logging/

**Reply by drmrbrewer** [reply]
Perfect, thanks.
